torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -86,6 +86,82 @@ void insertPrePackedConv2dOp(std::shared_ptr<Graph>& graph) {
   rewriter.runOnGraph(graph);
 }
 
+// Fusing Relu with Linear/Conv already implies that relu will
+// be done inplace. However, if the input of the relu was not to be
+// overwritten then it should not be fused.
+void fuseReluWithPackedOps(std::shared_ptr<Graph>& graph) {
+
+  SubgraphRewriter rewriter;
+
+  std::string linear_prepack_run_relu_fused = R"(
+    graph(%input, %weight, %bias, %output_min_max):
```

Comment:
What is output_min_max?  Isn't this supposed to appear in the pattern below?

Reply:
Graph fuser will complain if the graph being replaced with has a different signature. Thats why I had to keep it here.
 also realized, I think we will need to do constant pooling after this pass, because for every graph being replaced we are introducing the constant of '0'.